### ISSUE TITLE
Dashicons breaks section navigation height

### ISSUE THREAD

**Issue by fakrulislam** [themeum/kirki#1121]
### Issue description:

When using dashicons in sections the **navigation bar** height seems expands (see attachment). Is that normal?

![customizer-dashicons](https://cloud.githubusercontent.com/assets/15823216/18565250/dab2c41c-7bb0-11e6-9e4d-02bda4af7edb.png)
### Version used:

(Did you try using the develop branch from github? There's a chance your issue has already been adressed there)

Latest from WordPress.org (2.3.6)
### Using theme_mods or options?

theme_mods
### Code to reproduce the issue (config + field(s))

```
mytheme_Kirki::add_section( 'outlook_about_section', array(
    'title'                 => esc_attr__( 'About', 'mytheme' ),
    'priority'              => 10,
    'panel'                 => 'mytheme_front_page_panel',
    'icon'              => 'dashicons-align-left',
    'capability'            => 'edit_theme_options',
) );
```

**Comment by aristath** [comment]
Marked as a bug, will get it fixed and include in the next version.
Good catch!

**Comment by fakrulislam** [comment]
Hi @aristath , 
I found another issue here that you may consider to look at it also. When adding a panel icon, sections (under this panel) are automatically adding that icon too. If I use different icons for sections under that panel then two icon displays in the sections. Let me give you the screenshots. Im not so good in english, still learning besides coding so please hang with me.

Panel Icon:
`'icon'           => 'dashicons-admin-home'`

Section Icon:
`'icon'                     => 'dashicons-menu'`

Panel screenshot:
![panel](https://cloud.githubusercontent.com/assets/15823216/18607648/6d0b8846-7cf4-11e6-8225-b127f6736a5b.png)

Sections screenshot:
![sections](https://cloud.githubusercontent.com/assets/15823216/18607651/7b5fccb8-7cf4-11e6-9ac6-2dcfe888cc62.png)

**Comment by aristath** [comment]
Closing this one, will be fixed on #1125
